modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/ControlAcceleratorSupport.java

```diff
@@ -311,4 +332,42 @@ private static ReadOnlyObjectProperty<? extends Control> getControlProperty(Obje
 
         return null;
     }
+
+    // We need to store all the listeners added to each ObservableList so that we can remove them. For this we need
```

Comment:
could you convert this comment to a javadoc one please?
Eclipse shows the javadoc when one clicks on the type anywhere.